python/lbann/models/__init__.py

```diff
@@ -3,3 +3,6 @@
 from lbann.models.resnet import ResNet, ResNet18, ResNet34, ResNet50, ResNet101, ResNet152
 from lbann.models.roberta import RoBERTa
 from lbann.models.transformer import Transformer, TransformerEncoderLayer, TransformerDecoderLayer
+from lbann.models.subgraph.transformer import Transformer as TransformerSubGraph
```

Comment:
There is quite a bit of code duplication here, and the code is not up to date with the batched head implementation. What I'd suggest is that TransformerSubGraph would be a subclass of Transformer that sets up grid tags. We can discuss.

Reply:
Post-discussion note: we will leave this for another PR.